examples/tutorial4/tutorial/app.py

```diff
@@ -20,8 +21,7 @@ def startup(self):
         self.main_window.show()
 
     def fill_head(self):
-        with self.canvas.fill():
-            self.canvas.fill_style('rgba(149.0, 119, 73, 1)')
+        with self.canvas.fill(color='rgba(149.0, 119, 73, 1)'):
```

Comment:
Travertino colors are exposed now, so we can clean up this part of the API.

Reply:
OK, thanks, I didn't realize that. I'll make that update!